app/models.py

```diff
@@ -967,13 +967,24 @@ def available_sl_domains(self) -> [str]:
         """
         return [sl_domain.domain for sl_domain in self.get_sl_domains()]
 
-    def get_sl_domains(self) -> List["SLDomain"]:
-        query = SLDomain.filter_by(hidden=False).order_by(SLDomain.order)
-
-        if self.is_premium():
-            return query.all()
+    def get_sl_domains(self, domain_name: Optional[str] = None) -> list["SLDomain"]:
```

Comment:
`list` should be `List` I think

Reply:
In python 3.10 the standard types are included like `list`, `tuple`... https://docs.python.org/3.10/library/typing.html

Reply:
ah ok, didn't know that!